CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## Version 2.3.0-beta2
```

Comment:
Make sure all feature changes are captured in the changelog.
Then you can copy and paste from here to fill in the release description when the signed package is uploaded.